gst/nnstreamer/tensor_common.c

```diff
@@ -943,6 +944,16 @@ gst_tensors_config_from_structure (GstTensorsConfig * config,
               num_types, config->info.num_tensors);
         }
       }
+    } else if (format == _NNS_TENSOR_FORMAT_SPARSE) {
+      if (!gst_structure_get_int (structure, "num_tensors",
```

Comment:
discuss: requires num_tensors in caps ?

Reply:
No, it should be inferred from the number of memory. Thanks, I removed this part